### ISSUE TITLE
Add repair blacklist/whitelist

### ISSUE THREAD

**Issue by LuxusDarkangel** [hsyyid/EssentialCmds#126]
It accepts mod items right now, and that means a lot of troubles and bugs. For example, you can fill ic2 batterys with energy.

**Comment by MarkTwoFive** [comment]
Supporting mod items is a good thing IMO.
A settlement could be a black- or whitelist.

**Comment by kinggoesgaming** [comment]
:+1: for blacklisting /whitelisting

**Comment by hsyyid** [comment]
Actually, perhaps I can make it configurable instead? As suggested a black or whitelist or both for the repair.

**Comment by LuxusDarkangel** [comment]
Yeah, that's a better solution than not allowing any modded item to be repaired, but something has to be done to prevent things like IC2 batterys getting filled.

**Comment by MarkTwoFive** [comment]
Well at least for admins/higher grade moderators instantly filling batteries can also a good thing ;)
Maybe also add a permission for exemption from the white and/or blacklist
